src/bandersnatch/delete.py

```diff
@@ -45,11 +45,30 @@ async def delete_path(blob_path: Path, dry_run: bool = False) -> int:
     return 0
 
 
+async def delete_simple_page(
+    simple_base_path: Path, package: str, dry_run: bool = True
+) -> None:
+    if dry_run:
+        logger.info(f"[dry run]rm simple page of {package}")
+        return
+    simple_dir = simple_base_path / package
+    simple_index = simple_dir / "index.html"
+    simple_index.unlink(missing_ok=True)
+    if simple_dir.exists():
+        simple_dir.rmdir()
```

Comment:
I think this needs to be a little smarter if [hash-index](https://github.com/pypa/bandersnatch/blob/main/src/bandersnatch/default.conf#L50) is set.

- We should check if the hash is empty and delete if so maybe - Haven't looked into if non filesystem cares here or not ...